crypto/hd/hdpath_test.go

```diff
@@ -185,7 +184,8 @@ func TestDeriveHDPathRange(t *testing.T) {
 	}
 }
 
-func ExampleStringifyPathParams() { //nolint:govet
+/* THESE ARE EXAMPLE TESTS
+func ExampleStringifyPathParams() {
```

Comment:
Nope, we need to leave them uncommented so that it shows on https://pkg.go.dev/testing#hdr-Examples.
Please revert. What is govet saying? Maybe there is another way to fix it.

Reply:
ps: go is cruel -- so is rust, don't listen to the go haters